features/steps/partcad-cli/commands/init.py

```
@@ -0,0 +1,134 @@
+
+    start_time = time.time()
+    # Run the command in the shell
+    result = subprocess.run(
+        command,
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+        cwd=context.test_dir,
+        env=env,
+    )
```

Ruff (0.8.0)

Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)